model/config.go

```diff
@@ -215,22 +229,25 @@ func (c *CedarConfig) Find() error {
 		}
 	}
 
-	if err := c.find(); err != nil {
+	updates := make(chan interface{})
+	if err := c.createConfigWatcher(ctx, updates); err != nil {
```

Comment:
nit: rather than passing in the update channel, it seems a little cleaner if `createConfigWatcher` returns the update channel as a result to make it explicit that it's the producer and not the consumer, but I don't care that much.